internal/cmd/login.go

```diff
@@ -315,19 +319,28 @@ func loginToProvider(provider *api.Provider) (*api.LoginRequestOIDC, error) {
 	}, nil
 }
 
-func runSetupForLogin(client *api.Client) (string, error) {
-	setupRes, err := client.Setup()
+func runSignupForLogin(client *api.Client) (*api.LoginRequestPasswordCredentials, error) {
```

Comment:
User facing messages probably needs some refining. Right now it displays
```
$ infra login localhost
  Welcome to Infra. Set up your admin user.
? Email: test@example.com
? Password: ********
? Confirm Password: ********
  Logged in as test@example.com
```

Reply:
This is spot on, would just replace the `.` with `:`

```
$ infra login localhost Welcome to Infra. Set up your admin user:
```

Reply:
Updated
```
$ infra login localhost:
  Welcome to Infra. Set up your admin user:
? Email: test@example.com
? Password: ********
? Confirm Password: ********
  Logged in as test@example.com
```